packages/shared/src/styles/components/buttons.css

```
@@ -224,16 +240,7 @@
   color: var(--button-color, var(--button-default-color));
   & .icon {
     color: var(--button-icon, var(--button-default-icon, none));
-  }
-
-  &[aria-pressed="true"] {
-    --button-background: var(--button-pressed-background);
-    --button-icon: var(--button-hover-icon);
-  }
-
-  &:hover, &.hover, &:focus:focus-visible {
-    --button-icon: var(--button-hover-icon);
-  }
+  }  
```

Just reverting the placement here.

```suggestion
  }
  
  &[aria-pressed="true"] {
    --button-background: var(--button-pressed-background);
    --button-icon: var(--button-hover-icon);
  }

  &:hover, &.hover, &:focus:focus-visible {
    --button-icon: var(--button-hover-icon);
  }
```